src/views/v2/month/calendar-body/day/calendar-events/calendar-event/featured-image.php

```diff
@@ -9,12 +9,16 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.4
+ * @version TBD
  *
  */
 $event = $this->get( 'event' );
+$is_featured  = isset( $event->featured ) && $event->featured;
 
-if ( ! isset( $event->image ) ) { // @todo: use template tags for images here
+if (
+	! $is_featured
+	|| ! isset( $event->image )
+) { // @todo: use template tags for images here
```

Comment:
:warning: Comment refers to a TODO task "use template tags for images here"

<sub>hash: 1d98f1ba1b38300fa341934b523af58e</sub>